services/ark-api/ark-api/src/ark_api/api/v1/openai.py

```diff
@@ -87,22 +87,50 @@ def process_request_metadata(
                     "error": {
                         "message": f"Invalid Ark metadata: {str(e)}",
                         "type": "invalid_request_error",
-                        "code": "invalid_ark_metadata"
+                        "code": "invalid_ark_metadata",
                     }
-                }
+                },
             )
     # Ignore other metadata keys per OpenAI SDK pattern
     return None
 
 
+# TODO: Maybe we should start streaming first, wait for the first chunk/response
```

Comment:
can you double check this - either fix it or maybe raise a GH issue suggesting the improvement and then link to the GH issue rather than `TODO`